src/widgets/events/constants.ts

```diff
@@ -0,0 +1,13 @@
+import dayjs from 'dayjs';
+import { getActualData } from '@/shared/helpers/getActualData.ts';
+import { events } from 'data';
+
+export const displayedCardsQuantity = 2;
```

Comment:
I'd renamed it to be consistent with https://github.com/rolling-scopes/site/pull/572/files#diff-0f86dc252bc7e003acf812cf211873a2c752b94369fac3ac596669e44acdc0dfR2

Reply:
Rename to ` maxEventsCount `